daterangepicker.js

```diff
@@ -290,6 +291,11 @@
                 this.timePicker12Hour = options.timePicker12Hour;
             }
 
+            if (typeof options.autoApply === 'boolean') {
+                this.autoApply = options.autoApply;
+                this.container.find('.applyBtn, .cancelBtn').addClass('hide');
```

Comment:
<pre><code>                
if (this.autoApply) {
this.container.find(".applyBtn, .cancelBtn").addClass("hide");
                    if (this.singleDatePicker) {
                        this.container.find(".ranges").addClass("hide");
                    }
                }
</pre></code>